src/slide-util-ts.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { createGlobalStyle } from "styled-components";
+
+/**
+ * Create a theme style for a slide.
+ *
+ * @param {object} props Props.
+ * @param {string} props.id Slide execution id.
+ * @param {string | null} props.css Css as a string.
+ * @returns {object} The component.
+ */
+function ThemeStyles({ id, css }: { id: string; css?: string }): JSX.Element {
+  if (!css) {
+    return <></>;
+  }
+  const slideCss = css.replaceAll("#SLIDE_ID", `#${id}`);
```

Comment:
Why are we doing this (ie - please add a comment :))

Reply:
I have no clue, this is copy / paste code from slide-util.js only there to make TS happy.